src/app/design/frontend/base/default/template/shopgate/cloudapi/pipelineRequest.phtml

```diff
@@ -1,6 +1,8 @@
 <meta id="libshopgate">
 <script type="text/javascript">
+    let shopBaseUrl = '<?php echo $this->getShopBaseUrl() ?>';
+    let isInApp = '<?php echo Mage::helper('shopgate_cloudapi/request')->isShopgateRequest(); ?>';
     document.observe("dom:loaded", function () {
-        ShopgatePipelineRequest.fire(<?php echo $this->getJsonMethods() ?>)
+			ShopgatePipelineRequest.fire(<?php echo $this->getJsonMethods() ?>)
     });
```

Comment:
Please append a ; to this line as you are appending one to the other lines.

Reply:
Done